### PR TITLE
Feature applying style changes by design

### PR DESCRIPTION
**Pull request by Melissa-Martinez-XL**
## Ticket

* [Verify the current design and work with Isa](https://www.notion.so/xmartlabs/Verify-the-current-design-and-work-with-Isa-a4f28ffdce4a4ca09b5d748489bc0219?pvs=4)

## Type of change

* [X] Fix
* [ ] Story
* [ ] Chore

## Description of the change

I applied the comments of Isa (design) correcting the style. 
imagen
imagen

## Screenshot/Execution

https://user-images.githubusercontent.com/107500715/219402269-4cd28d46-a271-4276-92b3-b55030c03c1a.mp4

## Related PRs
Links to the related PRs

### CODE REVIEW

File: src/components/card/card.module.scss [hunk 1]
```diff
@@ -12,6 +12,7 @@
   padding-top: 56.6%;
   display: block;
   margin-bottom: 1.5rem;
+  width: 110%;
```
Comment (diegorosolino) [inline src/components/card/card.module.scss:15]: Is this correct? Why do you need more than 100%?

Comment (Melissa-Martinez-XL) [reply src/components/card/card.module.scss:15]: I was working a lot in this, I need the picture bigger than 100, but, If I make the container <article> bigger, the picture doesn't look like if I apply 110 to the picture. If you don't understand, I can try to explain this differently.

File: src/components/card/card.module.scss [hunk 2]
```diff
@@ -8,20 +8,18 @@
 }
 
 .imageContainer {
-  position: relative;
-  padding-top: 56.6%;
```
Comment (diegorosolino) [inline src/components/card/card.module.scss:12]: Why don't you use ratio 16:9 for image?

File: src/css/blog-post.module.scss [hunk 3]
```diff
@@ -76,8 +74,9 @@
 // Blog body
 
 .bodyPostContainer {
-  padding: 5rem 20rem 2rem 17rem;
+  padding: 5rem 38rem 2rem 17rem;
   text-align: justify;
+  font-family: 'work-sans';
```
Comment (diegorosolino) [inline src/css/blog-post.module.scss:79]: I think you can define this in text-styles

File: src/css/blog-post.module.scss [hunk 4]
```diff
@@ -114,12 +113,16 @@
 
   a {
     color: $blueTwo;
+    font-family: 'work-sans-bold';
```
Comment (diegorosolino) [inline src/css/blog-post.module.scss:116]: Same comment

File: src/css/blog-post.module.scss [hunk 5]
```diff
@@ -162,15 +165,45 @@
   }
 
   table {
-    @extend .text__paragraph__two__blueTwo;
+    font-family: 'gilroy', serif;
```
Comment (diegorosolino) [inline src/css/blog-post.module.scss:168]: Same comment

File: src/css/text-styles.scss [hunk 6]
```diff
@@ -106,10 +107,15 @@ $text-colors: (
 
 @mixin text__heading__two__separated {
   font-size: map-get($map: $text-sizes, $key: headingTwoSeparated);
-  line-height: map-get($map: $text-line-heights, $key: headingTwoSeparated);
+  line-height: map-get($map: $text-line-heights, $key: aside);
```
Comment (diegorosolino) [inline src/css/text-styles.scss:110]: typo error here

File: src/layouts/layout.module.scss [hunk 7]
```diff
@@ -9,12 +9,16 @@ body {
...
+  width: 100%;
+  height: 63.6rem;
 }
 
 .homeBanner {
   background-image: url("../../static/images/home-banner/home-banner-desktop.png");
   background-repeat: no-repeat;
   background-size: contain;
+  width: 100%;
+  height: 63.6rem;
```
Comment (diegorosolino) [inline src/layouts/layout.module.scss:21]: homeBanner and blogBanner have some styles in common, maybe you can define these styles at the same time:
 ```suggestion
 .homeBanner,
 .blogBanner {
  width: 100%;
  height: 63.6rem;
  background-repeat: no-repeat;
  background-size: contain;
```

File: src/components/card/card.module.scss (hunk 2 above)
Comment (Melissa-Martinez-XL) [reply src/components/card/card.module.scss:12]: I don't have any padding-top
image

File: src/css/text-styles.scss (hunk 6 above)
Comment (Melissa-Martinez-XL) [reply src/css/text-styles.scss:110]: Done

File: src/elements/containers.js [hunk 8]
```diff
@@ -55,6 +55,9 @@ export const StyledContainerNavBarXL = styled.div`
   width: 100%;
   height: 95px;
   justify-content: center;
+  background-color: white;
+  position: fixed;
+  z-index: 1;
```
[the remark this answers is not in the document]
Comment (Melissa-Martinez-XL) [reply src/elements/containers.js:60]: I can't use variables there, This: ${variable name} didn't work, and this: $variableName didn't work either

File: src/pages/{mdx.frontmatter__permalink}.js [hunk 9]
```diff
@@ -48,7 +48,7 @@ const BlogPost = ({ data }) => {
   
   return (
     <div onScroll={handleScroll}>
-      <SocialBlog className={disappearSocial ? styles.socialDisappear : styles.socialAppear} />
+      <SocialBlog className={disappearSocial ? styles.socialDisappear : styles.socialAppear} socialAtRight={true} />
```
[the remark this answers is not in the document]
Comment (Melissa-Martinez-XL) [reply src/pages/{mdx.frontmatter__permalink}.js:51]: done

File: src/components/card/card.module.scss (hunk 2 above)
Comment (Melissa-Martinez-XL) [reply src/components/card/card.module.scss:12]: Done?

File: src/css/variables.scss [hunk 10]
```diff
@@ -0,0 +1,2 @@
+// Index-Z
+$z-index-priority-small: 1;
```
Comment (diegorosolino) [inline src/css/variables.scss:2]: Remove this, you don't use it anywhere

Comment (Melissa-Martinez-XL) [reply src/css/variables.scss:2]: Done

File: src/css/blog-post.module.scss (hunk 5 above)
Comment (Melissa-Martinez-XL) [reply src/css/blog-post.module.scss:168]: Done

File: src/css/blog-post.module.scss [hunk 11]
```diff
@@ -114,12 +112,16 @@
 
   a {
     color: $blueTwo;
+    font-family: $link-blog-font;
+    font-weight: 600;
```
Comment (diegorosolino) [inline src/css/blog-post.module.scss:116]: I think this should be a type of text

File: src/css/text-styles.scss (hunk 6 above)
Comment (diegorosolino) [reply src/css/text-styles.scss:110]: The key es `aside`, is this correct?

Comment (Melissa-Martinez-XL) [reply src/css/text-styles.scss:110]: I don't think I use it anywhere. For me, I used it in a component that I had to delete, but, in this branch with main base, the text type does not appear.
image

**Review by diegorosolino — APPROVED** [review]

**Pull request merged**